src/config/networks.ts

```diff
@@ -2,16 +2,120 @@
 // SPDX-License-Identifier: Apache-2.0
 
 import { DefaultParams } from 'consts';
-import { ReactComponent as CreditcoinLogoSVG } from 'img/creditcoin_logo.svg';
-import { ReactComponent as PolkadotIconSVG } from 'img/polkadot_icon.svg';
-import { ReactComponent as PolkadotInlineSVG } from 'img/polkadot_inline.svg';
+import { ReactComponent as CreditcoinIconSVG } from 'img/ic_creditcoin.svg';
+import { ReactComponent as CreditcoinLogoSVG } from 'img/logo_creditcoin.svg';
+
 import type { Networks } from 'types';
 
 export const NetworkList: Networks = {
   creditcoin: {
     name: 'creditcoin',
     endpoints: {
-      rpc: 'ws://localhost:9944',
+      rpc: 'wss://rpc.mainnet.creditcoin.network/ws',
```

Comment:
I thought you had added an entry for "Local". have you decided against it ?

Reply:
Yes I opted to leave that out since it's only used by our internal team. I was concerned that it may be confusing to users.